.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Question on use of  graph sdk
+    url: https://github.com/microsoftgraph/msgraph-beta-sdk-java/discussions
```

Comment:
We will need to double check the discussions links across all PRs...

Reply:
```suggestion
    url: https://github.com/microsoftgraph/msgraph-sdk-java/discussions
```